### PR TITLE
052421 object tier new user guide

### PR DESCRIPTION
**Pull request by RRoten2004**
This is a brand-new markdown document that accesses 12 graphics to display in the article.

### CODE REVIEW

**Comment by ldsz89** [comment]
Need to remove the non markdown file

**Comment by ldsz89** [comment]
Is the ordered list to unordered list intentional here?

image

File: Lumen Network Storage/Getting Started/object-tier-new-user-guide [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  "attachments": [],
+  "contentIsHTML": false,
+  "sticky": true
+}}}
+#### Description
+This article outlines how customers can access and set up their Lumen Network Storage account. Instructions are provided for provisioning services. 
+### Accessing the Lumen Network Storage Portal
+
+**Creating Tenants and Users**
+The Lumen Network Storage Portal is a self-service interface into the most commonly used storage provisioning and configuration tasks. To access the portal, open https://storage.lumen.com in any web browser.
```
Comment (ldsz89) [inline Lumen Network Storage/Getting Started/object-tier-new-user-guide:14]: If you want that "Creating Tenant and Users" to be on a different line, you need another line under line 13.
image

Comment (RRoten2004) [reply Lumen Network Storage/Getting Started/object-tier-new-user-guide:14]: I made the change, staged it, committed it, and pushed it. If I missed a step, please advise.

**Review by ldsz89 — APPROVED** [review]

**Pull request merged**